backdrop/read/api.py

```diff
@@ -161,8 +161,6 @@ def data_group_key():
 
 
 @app.route('/data/<data_group>/<data_type>', methods=['GET', 'OPTIONS'])
-@limiter.limit(app.config.get('DATA_SET_RATE_LIMIT', '100/minute;5/second'),
```

Comment:
Can you also remove the `limiter` variable, which is now unused, and remove the library from `requirements.txt`?